src/window/building_info.c

```diff
@@ -101,43 +101,33 @@ static int get_height_id(void)
             return 5;
         }
         if (building_is_house(b->type) && b->house_population > 0) {
-            return 7;
+            return 5;
         }
 
         if (b->type >= BUILDING_SMALL_POND && b->type <= BUILDING_SENATOR_STATUE) {
             return 1;
         }
 
         switch (b->type) {
+//256px
```

Comment:
```suggestion
            // 256px
```

Use this formatting for all heights below please